OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -136,6 +114,28 @@ public function getOption($name)
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function setOptions(array $options)
```

Comment:
It was to be able to use options on a custom service, and be able to inject / manage them only when they are built. It indeed does not make any sense to alter them afterwards... I'm open to suggestions